modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+.Procedure
+
+. Choose the watchdog action. This defines how the watchdog proceeds when the operating system (OS) no longer responds.
+
++
+.Available actions
+[horizontal]
+`poweroff`:: The virtual machine (VM) powers down immediately.
+`reset`:: The VM reboots in place, and the guest OS cannot react. Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, you should avoid their use.
```

Comment:
`The VM reboots in place, and the guest OS cannot react.` - Remove comma.

`Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, you should avoid their use.` 
`timeout` > time out
"avoid" or "limit" their use? If the user needs to avoid this setting, why do we offer it to them? It would be helpful to explain this further, perhaps by telling the user when this should be used.

Reply:
@kbidarkar @stu-gott 
Can you provide additional info here?

Reply:
Liveness probes failing aren't a catastrophe at this point. The VM was already wedged and the watchdog kicked in. The impact is that from the user's point of view, it can take longer for the VM to be restarted.

How about this?

`reset`:: The VM reboots in place, and the guest OS cannot react. This can cause liveness probes to time out, which would make the VM take even longer to come back online. Use of this option is discouraged for this reason.

Reply:
To be clear, the VM takes longer because cluster-level protections will notice the liveness probe failed and forcibly reschedule it.

Reply:
I can work with that.